Sources/ConfidenceProvider/ConfidenceClient/ConfidenceClientOptions.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import Confidence
+
+public struct ConfidenceClientOptions {
+    public var credentials: ConfidenceClientCredentials
+    public var timeout: TimeInterval
+    public var region: ConfidenceRegion
+    public var initializationStrategy: InitializationStrategy
+
+    public init(
+        credentials: ConfidenceClientCredentials,
+        timeout: TimeInterval? = nil,
+        region: ConfidenceRegion? = nil,
+        initializationStrategy: InitializationStrategy = .fetchAndActivate
+    ) {
+        self.credentials = credentials
+        self.timeout = timeout ?? 10.0
+        self.region = region ?? .global
+        self.initializationStrategy = initializationStrategy
+    }
+}
+
+public enum ConfidenceClientCredentials {
```

Comment:
I know this is nothing new but what's the reason in having this `enum ConfidenceClientCredentials` instead of just a `String`?

Reply:
I am not sure either, but I wanted to keep it in order to maintain backwards compatibility in the builder:
```
OpenFeatureAPI.shared.setProvider(provider:
ConfidenceFeatureProvider.Builder(credentials: .clientSecret(secret: "mysecret"))
.build()
```